src/docfx/lib/HtmlUtility.cs

```diff
@@ -159,6 +163,32 @@ public static string TransformLinks(string html, Func<string, string> transform)
             return result.ToString();
         }
 
+        public static HtmlNode ExtractTitle(HtmlNode node)
+        {
+            foreach (var child in node.ChildNodes)
+            {
+                if (child.NodeType == HtmlNodeType.Comment)
+                {
+                    continue;
+                }
+
+                if (child.NodeType == HtmlNodeType.Text || string.IsNullOrWhiteSpace(child.OuterHtml))
+                {
+                    continue;
+                }
+
+                if (child.NodeType == HtmlNodeType.Element && child.Name == "h1")
```

Comment:
>  if (child.NodeType == HtmlNodeType.Element && child.Name == "h1") [](start = 14, length = 67)

we follow the v2 behavior here?

Reply:
yep, let me do it in another PR, want to keep one PR for one purpose